src/organisation/components/pba-numbers-form/pba-numbers-form.component.html

```diff
@@ -5,9 +5,11 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
 
-        <table class="govuk-table" *ngIf="updatePbaNumbers.hasDisplayPbaNumbers; else noPbaNumbersMessage">
+        <p *ngIf="noAccountsAdded">No PBA accounts added.</p>
```

Comment:
Just a quick query on the use of "PBA accounts"... was this agreed by the UX person at last week's demo? I think it was but just wanted to check they didn't suggest "PBA numbers" instead.

Reply:
Resolving comment since it's been confirmed the UX team were happy with "PBA accounts".